steps/bluejeans/meeting/create/main.go

```
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/stackpulse/steps-sdk-go/env"
+	"github.com/stackpulse/steps-sdk-go/step"
+	"github.com/stackpulse/steps/bluejeans/base"
+)
+
+type Args struct {
+	Username  string `env:"USERNAME"`
+	Password  string `env:"PASSWORD"`
+	RawToken  string `env:"BLUEJEANS_TOKEN"`
+	Title     string `env:"TITLE"`
+	SendEmail bool   `env:"SEND_EMAIL"`
+	Invitees  string `env:"INVITEES"`
+}
+type MeetingCreate struct {
+	args  Args
+	token *base.BluejeansToken
+}
+
+type output struct {
+	ApiResponse string `json:"api_response"`
+	MeetingUrl  string `json:"meeting_url"`
+	step.Outputs
+}
+
+func (s *MeetingCreate) Init() error {
+	err := env.Parse(&s.args)
+	if err != nil {
+		return err
+	}
+
```

maybe check here that we got username+password OR token and fail fast with invalid argument error